### ISSUE TITLE
Selection indicators in examples are missing aria-label

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#26209]
Make sure all examples that use checkboxes or radio indicators have an aria-label so that they are accessible to screen readers

https://react.fluentui.dev/?path=/docs/preview-components-table--default#cell-actions
https://react.fluentui.dev/?path=/docs/preview-components-datagrid--default
